src/client/sandbox/iframe.js

```diff
@@ -3,41 +3,40 @@ import NativeMethods from './native-methods';
 import Settings from '../settings';
 import * as DOM from '../utils/dom';
 import { syncServiceMsg } from '../transport';
-import { isMozilla, isIE, isWebKit } from '../utils/browser';
+import { isMozilla, isWebKit } from '../utils/browser';
 import { DOM_SANDBOX_OVERRIDE_DOM_METHOD_NAME } from '../../const';
-import { GET_IFRAME_TASK_SCRIPT as GET_IFRAME_TASK_SCRIPT_CMD } from '../../service-msg-cmd';
+import { getIframeTaskScript as GET_IFRAME_TASK_SCRIPT_CMD } from '../../service-msg-cmd';
 import { isSupportedProtocol, isIframeWithoutSrc } from '../utils/url';
 
+const IFRAME_WINDOW_INITED = 'hh_iwi_5d9138e9';
```

Comment:
I think we need some common pattern for all our DOM-integrated constants, with some naming convention with clear names, because currently it's pretty random and not clear. I think we can do it in separate PR after v1.0, can you fill the issue pls?